content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+
+* SSH ke mesin
+* Gunakan pengguna _root_ (e.g. `sudo su -`)
+* Jalankan perintah hasil keluaran `kubeadm init`. Sebagai contoh:
+
+```bash
+kubeadm join --token <token> <control-plane-host>:<control-plane-port> --discovery-token-ca-cert-hash sha256:<hash>
+```
+
+Jika kamu tidak memiliki _token_, kamu bisa mendapatkannya dengan menjalankan perintah berikut pada simpul _control-plane_:
+
+```bash
+kubeadm token list
+```
+
+Keluaran akan tampil seperti berikut:
+
+```console
+TOKEN                    TTL  EXPIRES              USAGES           DESCRIPTION            EXTRA GROUPS
+8ewj1p.9r9hcjoqgajrj4gi  23h  2018-06-12T02:51:28Z authentication,  The default bootstrap  system:
+                                                   signing          token generated by     bootstrappers:
+                                                                    'kubeadm init'.        kubeadm:
+                                                                                           default-node-token
+```
+
+Secara bawaan, _token_ akan kadaluarsa dalam 24 jam. Jika kamu menggabungkan simpul ke klaster setelah _token_ kadaluarsa,
+kamu dapat membuat _token_ baru dengan menjalankan perintah berikut pada simpul _control-plane_:
+
+```bash
+kubeadm token create
+```
+
+Keluaran akan tampil seperti berikut:
+
+```console
+5didvk.d09sbcov8ph2amjw
+```
+
+Jika kamu tidak memiliki nilai `--discovery-token-ca-cert-hash`, kamu bisa mendapatkannya dengan menjalankan perintah berantai berikut pada simpul _control-plane_:
+
+```bash
+openssl x509 -pubkey -in /etc/kubernetes/pki/ca.crt | openssl rsa -pubin -outform der 2>/dev/null | \
+   openssl dgst -sha256 -hex | sed 's/^.* //'
+```
+
+Keluaran yang diberikan kurang lebih akan ditampilkan sebagai berikut:
+
+```console
+8cb2de97839780a412b93877f8507ad6c94f73add17d5d7058e91741c9d5ec78
+```
+
+{{< note >}}
+Untuk menentukan _tuple_ IPv6 untuk `<control-plane-host>:<control-plane-port>`, alamat IPv6 harus be ditutup dengan kurung siku, sebagai contoh: `[fd00::101]:2073`.
+{{< /note >}}
+
+Keluaran yang diberikan kurang lebih akan ditampilkan sebagai berikut:
+
+```
+[preflight] Running pre-flight checks
+
+... (log output of join workflow) ...
+
+Node join complete:
+* Certificate signing request sent to control-plane and response
+  received.
+* Kubelet informed of new secure connection details.
+
+Run 'kubectl get nodes' on control-plane to see this machine join.
+```
+
+Beberapa saat kemudian, kamu akan melihat simpul tersebut pada keluaran dari `kubectl get nodes` ketika dijalankan pada simpul _control-plane_.
+
+### (Opsional) Mengendalikan klaster dari mesin selain simpul _control-plane_
+
+Untuk membuat kubectl bekerja pada mesin lain (e.g. laptop) agar dapat berbicara dengan
+klaster, kamu harus menyalin berkas kubeconfig administrator dari simpul _control-plane_
+ke _workstation_ seperti berikut:
```

Comment:
```suggestion
ke mesin tersebut seperti berikut:
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc